scripts/orthomcl-setup-database.pl

```diff
@@ -16,15 +16,16 @@
 
 my $default_config = "$script_dir/../etc/orthomcl.config.example";
 
-my $usage = "Usage: ".basename($0)." --user [user] --password [password] --host [host] --database [database]\n".
+my $usage = "Usage: ".basename($0)." --user [user] --password [password] --host [host] --database [database] --outfile [outfile]\n".
```

Comment:
Could you modify the usage statement here to describe in more detail what should go into these parameters.  That is, the user needs create permissions for the database, the database is the one that needs to be created (so it doesn't need to exist), etc?